sql/parser/sql.y

```diff
@@ -830,10 +835,26 @@ var_value:
   }
 
 iso_level:
-  READ UNCOMMITTED {}
-| READ COMMITTED {}
-| REPEATABLE READ {}
-| SERIALIZABLE {}
+  READ UNCOMMITTED
```

Comment:
I could remove them, but I think they are part of the SQL standard. I'll add a comment.